.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "data"]
 	path = data
 	url = git@github.com:alphagov/pay-cardid-data.git
+    branch = master
```

Comment:
https://git-scm.com/docs/gitmodules
According to the documentation is always defaults to `master`